packages/plugins/content/slate/src/hooks.js

```diff
@@ -61,7 +63,18 @@ export const defaultPlugins = [
 export const lineBreakSerializer = {
   deserialize(el: any) {
     if (el.tagName.toLowerCase() === 'br') {
-      return { kind: 'text', text: '\n' }
+      return { object: 'text', text: '\n' }
```

Comment:
Where is this serializer taken from?

Reply:
From docs, just needed to modify it to support the actual text nodes